resources/js/Components/Navigation.svelte

```diff
@@ -1,19 +1,163 @@
 <script>
     import { inertia } from "@inertiajs/svelte";
     import Logo from "@components/Logo.svelte";
+    const links = {
+        discord: "https://discord.gg/tCg94qy",
+        wiki: "https://wiki.abidanarchive.com",
+    };
+    let showMenu = false;
+
+    function toggleNavbar() {
+        showMenu = !showMenu;
+    }
 </script>
 
-<nav
-    class="fixed z-50 top-0 left-0 h-16 w-full bg-midnight border-l-5 border-abidan-400 text-white drop-shadow-md"
->
-    <div class="px-4 sm:px-6">
-        <div class="flex justify-between h-16 w-full">
-            <a use:inertia href={route("home")}>
-                <div class="flex flex-row h-full justify-start items-center">
-                    <Logo />
-                    <h1 class="shrink-0 text-3xl px-2">Abidan Archive</h1>
+<div>
+    <div class="fixed top-0 left-0 w-full h-auto z-50 bg-gray-800">
+        <nav
+            class="container px-6 py-6 mx-auto md:flex md:justify-between md:items-center"
+        >
+            <div class="flex items-center justify-between">
+                <a use:inertia href={route("home")}>
+                    <div
+                        class="flex flex-row h-full justify-start items-center"
+                    >
+                        <Logo />
+                        <h1 class="shrink-0 text-3xl px-2">Abidan Archive</h1>
+                    </div>
+                </a>
+                <!-- Mobile menu button -->
+                <!-- svelte-ignore a11y-click-events-have-key-events -->
+                <div on:click={toggleNavbar} class="flex md:hidden">
+                    <button
+                        type="button"
+                        class="text-white-100 hover:text-white-400 focus:outline-none focus:text-white-400"
+                    >
+                        <svg
+                            xmlns="http://www.w3.org/2000/svg"
+                            fill="none"
+                            viewBox="0 0 24 24"
+                            stroke-width="1.5"
+                            stroke="currentColor"
+                            class="w-8 h-8"
+                        >
+                            <path
+                                stroke-linecap="round"
+                                stroke-linejoin="round"
+                                d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+                            />
+                        </svg>
+                    </button>
                 </div>
-            </a>
-        </div>
+            </div>
+
+            <!-- Mobile Menu open: "block", Menu closed: "hidden" -->
+            <div
+                class="flex-col mt-8 lg:text-xl space-y-4 md:flex md:space-y-0 md:flex-row md:items-center md:space-x-6 lg:space-x-10 md:mt-0 {showMenu
+                    ? 'flex'
+                    : 'hidden'}"
```

Comment:
Let's use conditional classes here from svelte,
https://svelte.dev/tutorial/classes

or clsx (already installed)
https://www.npmjs.com/package/clsx
which ever you prefer.

Reply:
Done using class directives from svelte.